src/main/kotlin/io/github/anki/anki/controller/dto/CardDto.kt

```diff
@@ -1,14 +1,22 @@
 package io.github.anki.anki.controller.dto
 
+import jakarta.validation.constraints.NotBlank
+
 data class NewCardRequest(
-    var deckId: String,
-    var cardKey: String,
-    var cardValue: String,
+    @field:NotBlank(message = "should not be blank")
```

Comment:
Why you don't like default message?
<img width="602" alt="image" src="https://github.com/user-attachments/assets/a2bf004c-98ca-4a37-a2a5-29c678c64bf1">

Reply:
i didnt know it has good default message, will use it